src/main.c

```diff
@@ -247,10 +344,93 @@ void WriteInterfaces()
     free(InterfaceArray);
 }
 
+void CreateVmSwitchPacketFragment(PEVENT_RECORD ev)
+{
+    if (ev->EventHeader.EventDescriptor.Id != tidVMSwitchPacketFragment) {
+        printf("CreateVmSwitchPacketFragment called but EventId != tidVMSwitchPacketFragment\n");
+        return;
+    }
+
+    wchar_t buffer[8192];
```

Comment:
Where does 8192 come from?

Reply:
I have to confess I picked a random value big enough to store the result of TdhGetProperty() call...